pkg/check/table_structure.go

```diff
@@ -374,7 +374,7 @@ func (c *ShardingTablesCheck) checkAutoIncrementKey(instance, schema, table stri
 		if !hasMatchedRule {
 			r.State = StateFailure
 			r.ErrorMsg = fmt.Sprintf("instance %s table `%s`.`%s` of sharding %s have auto-increment key, would conflict with each other to cause data corruption", instance, schema, table, c.name)
-			r.Instruction = "please set column mapping rules for them, or handle it by yourself"
+			r.Instruction = "please handle it by yourself, read https://pingcap.com/docs-cn/dev/reference/tools/data-migration/usage-scenarios/best-practice-dm-shard/#自增主键冲突处理 for more detail"
```

Comment:
also put English document link here？

Reply:
don't have english doc now, update it later?

Reply:
add english doc firstly, some foreign users need it

Reply:
ok

Reply:
should we link to `dev` or `v3.0`?

Reply:
If the user want to _continue_ after _handled it by themself_, what should they do in _dmctl_?

Reply:
because dm's version is independent, so I think dev is better.

Reply:
also add a tip for how to do? but this is a pkg, infact I think add a dm's link here is not very suitable.